agent/src/main/scala/za/co/absa/atum/agent/AtumContext.scala

```diff
@@ -0,0 +1,66 @@
+package za.co.absa.atum.agent
+
+import org.apache.spark.sql.DataFrame
+import za.co.absa.atum.agent.model.{MeasureResult, Measurement}
+
+/**
+ *  AtumContext: This class provides the methods to measure Spark `Dataframe`. Also allows to add/edit/remove measures.
+ *  @param measurements: A sequences of measurements.
+ */
+
+case class AtumContext(measurements: Map[String, Measurement] = Map()) {
```

Comment:
Measurement don't (shouldn't) have name. Perhaps `Set` is enough. Or `Seq` if order matters, but IMHO it doesn't.